cmds/core/cpio/cpio_test.go

```diff
@@ -26,13 +25,9 @@ type dirEnt struct {
 func TestCpio(t *testing.T) {
 	debug = t.Logf
 	// Create a temporary directory
-	tempDir, err := ioutil.TempDir("", "TestCpio")
-	if err != nil {
-		t.Fatalf("cannot create temporary directory: %v", err)
-	}
-	defer os.RemoveAll(tempDir)
+	tempDir := t.TempDir()
```

Comment:
> Can you clarify why this err is being added?

@rjoleary This is because I have used `t.TempDir()` to create the temporary directory instead of `ioutil.TempDir()`, and hence the `err` variable is no longer defined here.